OpenTabletDriver.Web/Views/Wiki/FAQ/Linux.cshtml

```diff
@@ -226,7 +226,8 @@
                 Add this block below the first 2 lines of the file, then save and exit
                 <codeblock class="mt-2" language="ini">
                     [Service]
-                    Environment=WAYLAND_DISPLAY=:0
+                    ExecStart=
```

Comment:
Is there a reason for this empty extra `ExecStart`?

Reply:
The empty ExecStart is to clear the value. Otherwise when you run it, it will run both the default start command and the custom one which makes it throw an error.

Reply:
That's pretty weird behavior, makes me wonder if that's what it's supposed to do.